CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
+* Updated documentation about testing
+* Syn library is [0.0.2](https://github.com/bitovi/syn/tree/v0.0.2)
+* `setDesiredCapabilities` will throw an exception if you will try to set capabilities on already started session
+* Added `getDesiredCapabilities` to allow fetch capabilities
+* Minors improvements to driver when running against W3C compatible driver
+ 
+###### Testsuite
+* Adding testing on PHP 7.1, 7.2, 7.3 and 7.4beta
+
+###### Why it's a minor version bump
```

Comment:
Please remove this section.

---

After releasing we won't change version to 2.0.0 anyway.

Reply:
Mostly it's an answers why dropping 5.3 is not a BC or major bump.